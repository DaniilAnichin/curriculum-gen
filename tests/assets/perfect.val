Violations of Lectures (hard) : 0
Violations of Conflicts (hard) : 0
Violations of Availability (hard) : 0
Violations of RoomOccupation (hard) : 0
Cost of RoomCapacity (soft) : 0
Cost of MinWorkingDays (soft) : 0
Cost of CurriculumCompactness (soft) : 0
Cost of RoomStability (soft) : 0

Summary: Total Cost = 0
