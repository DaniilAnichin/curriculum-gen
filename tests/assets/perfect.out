SceCosC B 1 1
SceCosC B 2 1
SceCosC B 3 2
ArcTec B 1 3
ArcTec B 2 3
ArcTec B 3 1
TecCos B 1 2
TecCos B 2 2
TecCos B 3 0
TecCos B 4 1
TecCos B 4 2
Geotec A 1 1
Geotec A 2 0
Geotec A 2 1
Geotec A 3 1
Geotec A 4 0
